packages/cli/src/commands/mock.ts

```diff
@@ -44,7 +44,7 @@ const mockCommand: CommandModule = {
       jsonSchemaFakerFillProperties,
     };
 
-    return runPrismAndSetupWatcher(createPrism, options);
+    return runPrismAndSetupWatcher(createPrism, options) as Promise<void>;
```

Comment:
Type error was - I am sure there is a better way to solve this
```
Type '(parsedArgs: ArgumentsCamelCase<{}>) => Promise<unknown>' is not assignable to type '(args: ArgumentsCamelCase<{}>) => void | Promise<void>'.
      Type 'Promise<unknown>' is not assignable to type 'void | Promise<void>'.
        Type 'Promise<unknown>' is not assignable to type 'Promise<void>'.
          Type 'unknown' is not assignable to type 'void'.

```

Reply:
is this indicative that the caller of this function won't get what it needs? or it just doesn't care?

Reply:
thanks for walking through the types changes with me, you should see our changes reflected in the most recent commit.